python/ray/serve/http_proxy.py

```diff
@@ -209,6 +209,7 @@ def __init__(self, controller_name: str):
         # controller instance this proxy is running in.
         ray.serve.api._set_internal_replica_context(None, None,
                                                     controller_name, None)
+        controller_namespace = ray.serve.api._get_controller_namespace()
```

Comment:
Given that the HTTP proxy is always started by the controller, I think we can just always use the current namespace here. This logic should work do the same thing though.

Reply:
Yes, I think the HTTP proxy is being started in the current namespace but because it is getting the controller actor, I'm passing in the `controller_namespace` also